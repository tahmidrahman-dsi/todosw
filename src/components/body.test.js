import React from 'react';
import Body from './body';
import renderer from 'react-test-renderer';

describe('Body Component', () => {
  it('matches Snapshot', () => {
    const tree = renderer.create(<Body/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
