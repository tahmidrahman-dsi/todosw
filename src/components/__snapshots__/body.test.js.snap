// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body Component matches Snapshot 1`] = `
<div>
  <div
    className="react-tabs"
    data-tabs={true}
    onClick={[Function]}
    onKeyDown={[Function]}
  >
    <ul
      className="react-tabs__tab-list"
      role="tablist"
    >
      <li
        aria-controls="react-tabs-1"
        aria-disabled="false"
        aria-selected="true"
        className="react-tabs__tab react-tabs__tab--selected"
        id="react-tabs-0"
        role="tab"
        tabIndex="0"
      >
        Ball drop Animation
      </li>
      <li
        aria-controls="react-tabs-3"
        aria-disabled="false"
        aria-selected="false"
        className="react-tabs__tab"
        id="react-tabs-2"
        role="tab"
        tabIndex={null}
      >
        Flux Test
      </li>
    </ul>
    <div
      aria-labelledby="react-tabs-0"
      className="react-tabs__tab-panel react-tabs__tab-panel--selected"
      id="react-tabs-1"
      role="tabpanel"
    >
      <div
        className="stage"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="container"
        >
          <h1>
            Hover to start animation
          </h1>
        </div>
        <div
          className="ball"
        />
      </div>
    </div>
    <div
      aria-labelledby="react-tabs-2"
      className="react-tabs__tab-panel"
      id="react-tabs-3"
      role="tabpanel"
    />
  </div>
</div>
`;
